From 6b59083b420d12baf688669660cb8f0b1838e538 Mon Sep 17 00:00:00 2001
From: Natanael Copa <ncopa@alpinelinux.org>
Date: Mon, 23 Dec 2024 22:46:33 +0100
Subject: [PATCH 1/2] setup-wayland-base: explicit start dbus

Seems like elogind fails to properly pull in dbus. Work around it by
explicitly start dbus first.
---
 setup-wayland-base.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup-wayland-base.in b/setup-wayland-base.in
index 267c835..53a4a9e 100644
--- a/setup-wayland-base.in
+++ b/setup-wayland-base.in
@@ -30,7 +30,7 @@ apk add elogind polkit-elogind eudev "$@"
 
 setup-devd udev
 
-for service in elogind polkit; do
+for service in dbus elogind polkit; do
 	rc-service "$service" start
 	rc-update add "$service"
 done
-- 
2.47.1

